### ISSUE TITLE
[Content styleguide] The content isn't written consistently across static pages, tools, and devices.  (05.11.1)

### ISSUE THREAD

**Issue by chaseakirby** [department-of-veterans-affairs/va.gov-team#39477]
#### VFS team name
Authenticated Experience

#### VFS product name
My VA Dashboard

#### VFS feature name
Payment Information

#### Point of Contact/Reviewers
Cheryl Evans (@CherylEvans) - Content

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
The content isn't written consistently across static pages, tools, and devices. 

### Issue Details
Benefit Payments and Debts, and Benefit payments, debts, and copays, all contain Type: whereby specific types of benefit payments origin is noted.  Debts may does not have a Type: listing, making it inconsistent with Payments Type (e.g., compensation & pension)

### VA.gov Experience Standard
[Category Number 05, Issue Number 11](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.3.2_A

---

### Platform Recommendation
Include Type: under Debts for consistency to Benefit Payments and Debt, since these 2 are categorized together and live on the same hierarchal plane.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by hdjustice** [comment]
4/6/22 - Discussion with Angela & Samara - We are working on the organization of the section for future iterations

**Comment by aagosto90** [comment]
Same comment as the other ticket: This feedback is for the previous version of payment info "Benefit Payments and Debts" which is no longer live and has been replaced by separate sections for Outstanding Debts and Benefit Payments. We've handled all must feedback for the new version prior to launch so I think this ticket can be closed @shiragoodman @ajakabcin @ericasauv09

**Comment by shiragoodman** [comment]
Thank you @aagosto90 - I agree. Closing!
